documentation/leo/03_language.md

```diff
@@ -423,12 +424,24 @@ If the value at `addr` does not exist, then the program will fail to execute.
 A get command that initializes the mapping in case of failure, e.g.
 `let current_value: u64 = Mapping::get_or_init(counter, addr, 0u64);`
 Gets the value stored at `addr` in `counter` and stores the result in `current_value`.
-If the key is not present, `0u64`  is stored in `counter` and stored in `current_value`.
+If the key is not present, `0u64` is stored in `counter` and stored in `current_value`.
 
 #### set
 A set command, e.g. `Mapping::set(counter, addr, current_value + 1u64);`
 Sets the `addr` entry as `current_value + 1u64` in `counter`.
 
+## For Loops
+
+For Loops are declared as `for {loop variable} in {lower bound}..{upper bound}`.  Currently, the
+loop variable supports unsigned integer types u8, u16, u32, or u64. The lower bound must be
+less than the upper bound. Nested loops are supported.
+
+```leo
+for {loop variable} in {lower bound}..{upper bound} {
+    // Loop body
+}
+```
+
```

Comment:
You should add a code snippet of a loop that actually compiles.
People will want to copy and paste code from the docs into their program to play around with the feature.